### PR TITLE
Configuration

### PR DESCRIPTION
**Pull request by JoshVanL**
```release-note
NONE
```

### CODE REVIEW

**Comment by JoshVanL** [comment]
@munnerz Should be good to have a look

File: docs/configuration/acme/dns01/index.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+---
+title: "DNS01"
+linkTitle: "DNS01"
+weight: 30
+type: "docs"
+---
+
+# Configuring DNS01 Challenge Provider
```
Comment (munnerz) [inline docs/configuration/acme/dns01/index.md]: This file should be named `_index.md` else it won't be treated as the index by Hugo

File: docs/configuration/acme/index.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
+---
+title: "ACME"
+linkTitle: "ACME"
+weight: 30
+type: "docs"
+---
+
+# ACME Issuer
```
Comment (munnerz) [inline docs/configuration/acme/index.md]: Should also be _index.md

File: docs/configuration/_index.md [hunk 3]
```diff
@@ -1,6 +0,0 @@
----
-title: "Configuration"
-linkTitle: "Configuration"
-weight: 30
-type: "docs"
----
```
Comment (munnerz) [inline docs/configuration/_index.md]: This shouldn't be deleted

**Pull request merged**